ktor-server/ktor-server-tests/jvmAndNix/test/io/ktor/tests/server/plugins/CookiesTest.kt

```diff
@@ -377,3 +383,49 @@ class ParserServerSetCookieTest {
         assertEquals("/", parsed.path)
     }
 }
+
+class SecureCookiesTest {
+    @Test
+    fun `consider wss and https to be secure contexts`() {
+        testApplication {
```

Comment:
can be inlined as `fun test() = testApplication { ... } `

Reply:
inlined